contents/handbook/people/team-structure/infrastructure.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Team Infrastructure
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- James Greenhill (Team lead, Data/Infra Engineer)
+- Guido Iaquinti (Site Reliability Engineer)
+- Harry Waye (Full Stack Engineer)
+
+## Mission Statement
+Make deploying, scaling, and managing PostHog easy, fast, and reliable.
+
+## Q3 2022 Goals
+* **Proposed Objective**:
+    * Lots of customers are self-serving PostHog Cloud in the EU
+* **Proposed Key Results**:
+    * >100 organizations using EU hosted PostHog Cloud
+    * All deployment methods are support the latest stable versions of dependencies (so we can leverage the benefits of later versions)
```

Comment:
```suggestion
    * All deployment methods support the latest stable versions of dependencies (so we can leverage the benefits of later versions)
```
Typo fix, but this feels arbitrary. I'd like to have concrete reasons _why_ we should support any new version of anything. Specific security patches, performance, features? Like we don't want to support CH 22.6 because it's a mess right now, but would satisfy this goal.

Reply:
Agree, this is fairly arbitrary, is there a better way to highlight the critical dependencies for us to upgrade in Q3 as key results (e.g. like we needed to do a previous clickhouse upgrade).

@guidoiaquinti already mentioned "Kubernetes v1.25" below, and since we have some customers who rely on bleeding edge versions this could be critical.

Would be great if we could enumerate the biggest ones specifically in this key result.